### DISCUSSION TITLE
XCode workspace with more framework projects

### DISCUSSION THREAD

**Discussion by amatig** [peripheryapp/periphery#403] [Q&A]
Hi,
I am using a workspace with in multiple frameworks.
The app is one of them.
I am using the script on the workspace and I have specified the schema and target but it doesn't find the the frameworks.
The are all at the same level in the workspace.

Workspace:
- MyApp.xcodeproject
- ModuleA.xcodeproject
- ModuleB.xcodeproject

They are all at the same level, not nested.

It says: error: (PBXProjError) Cannot calculate full path for file element "Modules/ModuleA/ModuleA.xcodeproj" in source root: "MyProject"

**Comment by ileitch** [comment]
What's the location on disk of the framework `.xcodeproj`? Can you please paste the contents of the `contents.xcworkspacedata` file that lives inside the `.xcworkspace` folder (it shouldn't contain anything sensitive). Or even better, is the project public?
